### ISSUE TITLE
lint: should we add some eslint-plugins?

### ISSUE THREAD

**Issue by jalezi** [subhoghoshX/laureate#45]
I wonder if we should use some eslint plugins? E.g. `eslint-plugin-jsx-a11y`

**Comment by subhoghoshX** [comment]
Hey, thanks. I think NextJS already comes with ESLint integrated.

**Comment by jalezi** [comment]
nope 😞 

without:
![Screenshot 2022-10-08 at 18 45 35](https://user-images.githubusercontent.com/44704999/194718335-7e99259a-e028-4304-9111-995e022ab5f5.png)

with:
![Screenshot 2022-10-08 at 18 44 34](https://user-images.githubusercontent.com/44704999/194718361-dfc172b4-af7a-44c0-865e-f42e5903359f.png)

**Comment by subhoghoshX** [comment]
Yeah, ESLint is there, but not the plugin you mentioned. Not sure how much necessary it is.

**Comment by jalezi** [comment]
If it's fine with you, it's fine with me. 😄

**Comment by subhoghoshX** [comment]
Yeah, let's add it, a little bit of accessibility might be worth it.

**Comment by jalezi** [comment]
Actually [Next.js](https://nextjs.org/docs/accessibility#linting) uses `eslint-plugin-jsx-a11y` but not all rules as I understand.

**Comment by subhoghoshX** [comment]
> Actually [Next.js](https://nextjs.org/docs/accessibility#linting) uses `eslint-plugin-jsx-a11y` but not all rules as I understand.

Not sure, what's the point of adding it then. I have to look into it later.
